Loger/Starter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Loger
+{
+    static class Starter
+    {
+        private static int _logCount = 100;
```

Comment:
Its good practice to write explicitly public, internal modifiers near class or method.